migration/v2.18.0-ck8s1-v2.19.x-ck8s1/upgrade-cluster.md

```diff
@@ -25,3 +25,5 @@
 ```
 
 **NOTE**: If you are running [Compliant Kubernetes Apps](https://github.com/elastisys/compliantkubernetes-apps). Make sure you're running a version that has support for conatinerd.
+
+1. On openstack environments, export `CK8S_CONFIG_PATH`, `OS_USERNAME` and `OS_PASSWORD`, and run `migrate-terraform-openstack.sh`.
```

Comment:
Not sure about putting this here, since if you checkout `v2.19.0-ck8s1` then it will not show up.
Could you also/instead put it in the migration doc to `2.19.0-ck8s2`? https://github.com/elastisys/compliantkubernetes-kubespray/blob/main/migration/v2.19.0-ck8s1-v2.19.0-ck8s2/upgrade-cluster.md
I think we will need to realse that patch soon anyway.